Aigamo.ResXGenerator.Tests/SettingsTests.cs

```diff
@@ -273,7 +274,6 @@ public void FileSettings_RespectsGlobalDefaults()
 		fileOptions.StaticMembers.Should().Be(false);
 		fileOptions.PublicClass.Should().Be(true);
 		fileOptions.PartialClass.Should().Be(true);
-		fileOptions.IsValid.Should().Be(true);
```

Comment:
Just out of curiosity, why did you remove this assertion?

Reply:
Ah, I should have left a comment on it. It is because it is duplicating the last assertion in this test.